src/Tizen.NUI/src/public/BaseComponents/TextEditor.cs

```diff
@@ -945,6 +945,22 @@ public string PlaceholderText
             }
         }
 
+        /// <summary>
+        /// The Selected Text property.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
```

Comment:
please remove this <since_tizen> tag if the API is hidden-API.
please remove **"/// This will be public opened in tizen_6.0 after ACR done ..."** comment, because it's too late to add this API in tizen_6.0.